server.js

```diff
@@ -1,13 +1,56 @@
 const express = require('express');
+const redis = require('redis');
+const bodyParser = require('body-parser');
+
 
 const app = express();
+const redisClient = redis.createClient();
```

Comment:
Pass in the redis host address and the port, there is no guarantee that the application server and the redis server will be on the same machine. In fact, it’s not considered a good practice to keep the database and the application on the same machine.